### PR TITLE
Release html minify [issue #32]

### PR DESCRIPTION
**Pull request by Kirill-frontend**
Features:
The functionality was made with minification of the html file,
when using different scripts
-----------------------------------
Changelog: 
Edit package.json (in scripts)
Install two packages (cross-env, eleventy-plugin-files-minifier)
Edit .eleventy.js
---------------------------------------
P.S. 
I'm new. I would be grateful for advice

### CODE REVIEW

File: .eleventy.js [hunk 1]
```diff
@@ -1,8 +1,12 @@
 const markdownIt = require('markdown-it');
 const emoji = require('markdown-it-emoji');
+const eleventyPluginFilesMinifier = require('@sherby/eleventy-plugin-files-minifier');
 
 const { isAfter, isBefore, isToday, format } = require('date-fns');
 
+const isProduction = process.env.NODE_ENV || false;
```
Comment (msutkowski) [inline .eleventy.js:7]:
```suggestion
const isProduction = process.env.NODE_ENV === 'production';
```

File: .eleventy.js [hunk 2]
```diff
@@ -56,6 +60,10 @@ module.exports = (eleventyConfig) => {
 		return `${format(new Date(date), 'MMM d, yyyy p')} PST`;
 	});
 
+	if (isProduction === 'production') {
```
Comment (msutkowski) [inline .eleventy.js:63]:
```suggestion
	if (isProduction) {
```

**Review by msutkowski — CHANGES_REQUESTED** [review]
Looks good! Just a small suggestion.

**Comment by BenDMyers** [comment]
@Kirill-frontend This looks great! Thank you so much for taking this on!

I agree with @msutkowski's review above. Would you be willing to make that change for us?

**Comment by Kirill-frontend** [comment]
Yes, i can.

**Comment by BenDMyers** [comment]
This looks great, @Kirill-frontend! I'm going to go ahead and merge this in now. Thank you so much for pulling in this work!

**Review by BenDMyers — APPROVED** [review]

**Comment by chantastic** [comment]
Thanks @Kirill-frontend! This is a really great optimization PR 🙏

**Comment by BenDMyers** [comment]
@all-contributors Please add @Kirill-frontend for code.

**Pull request merged**